code/modules/telesci/gps.dm

```diff
@@ -54,7 +56,7 @@ GLOBAL_LIST_EMPTY(GPS_list)
 	addtimer(CALLBACK(src, PROC_REF(reboot)), EMP_DISABLE_TIME)
 
 /obj/item/gps/AltClick(mob/user)
-	if(ui_status(user, GLOB.inventory_state) != STATUS_INTERACTIVE)
+	if(ui_status(user, GLOB[tgui_state]))
```

Comment:
erghh, this is weird syntax for GLOB variables, not sure if this is the right way to do this
@AffectedArc07 @SteelSlayer @farie82  one of you nerds look at this

Reply:
I can remove this, I changed how the gps module worked

Reply:
I wouldn't remove it yet, I just don't know if this is how we should be accessing globs or not